example/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "example",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "react": "^16.5.2",
```

Comment:
You can add your library as a dependency here!

`"react-login-page": "https://github.com/AndrewZamora"`

Some notes here:

1.) This changes the imports below `/example` to from `../../src/Login` to `react-login-page`
2.) It would be good to make sure that the "name" in the `./package.json` is maybe better, like: `react-login-form` or `@AndrewZamora/react-login` (`@AndrewZamora` being the npm username!)
3.) You would probably need to have a compiled version of the `src` folder in the repository for this to work.
4.) As-is the example page wouldn't work, you need to trick a bit to make the local dev environment work properly.